src/analysis/souffle/tests/sql_verifier_dl_unit_tests/sql_output_result/isOperation.facts

```diff
@@ -0,0 +1,8 @@
+["sql", "arbitrary", "path1", nil, nil]
```

Comment:
Can a  `.facts` file have comments? If so, can you add an IR representation for these operations? Are all of these facts operations associated with a single query?

Reply:
Comments would be very nice, but I don't think they are supported by Souffle. I don't see anything in the documentation about them anyway.

Yes, all of the facts in the `isOperation.facts` file are `Operation`s associated with a single query.